scripts/start_docker_bblfsh.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# CI convenience script to start a bblfsh server in Docker container.
+# Skips starting server for jobs that do not need it i.e stylecheck
+
+E_BAD_DOCKER=2
+image="bblfsh/bblfshd:v2.4.2"
+
+set -ev
+
+if [[ -z "${STYLE_CHECK}" ]]; then
```

Comment:
Maybe we should find a better way to skip docker images? For instance we will have at some point lint for python, scala, and other possible new tests that need a different env.
Would it work if in travis.yml we moved the `before_script:` inside each matrix entry? I would be more verbose, but easier to spot where we are launching docker.
I'm not sure about the best option, but it's something worth looking into.

Reply:
current check is copy-past for a script we already use. But I agree is need something better. Maybe also use docker-compose as proposed by smola. Out of scope for this PR though.

@carlosms if you have good ideas for improvement travis, please create an issue with a description because I didn't really think about it yet.